node/service/src/chain_spec/frequency_paseo_local.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(missing_docs)]
+use common_runtime::constants::{FREQUENCY_TESTNET_TOKEN, TOKEN_DECIMALS};
+use frequency_runtime::Ss58Prefix;
+use polkadot_service::chain_spec::Extensions as RelayChainExtensions;
+use sc_service::ChainType;
+
+use super::{get_properties, Extensions};
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec = sc_service::GenericChainSpec<Extensions>;
+
+// Generic chain spec, in case when we don't have the native runtime.
+pub type RelayChainSpec = sc_service::GenericChainSpec<RelayChainExtensions>;
+
+#[allow(clippy::unwrap_used)]
+/// Generates the Frequency Paseo chain spec from the raw json
+pub fn load_frequency_paseo_spec() -> ChainSpec {
+	ChainSpec::from_json_bytes(
+		&include_bytes!("../../../../resources/frequency-paseo.raw.json")[..],
+	)
+	.unwrap()
+}
+
+// TODO: Remove once on a Polkadot-SDK with Paseo
+#[allow(clippy::unwrap_used)]
+/// Generates the Paseo Relay chain spec from the json
+pub fn load_paseo_spec() -> RelayChainSpec {
+	RelayChainSpec::from_json_bytes(&include_bytes!("../../../../resources/paseo.json")[..])
```

Comment:
nit: if we already have `Paseo-Local Relay chain spec` why do we need this? Doesn't seem to be used.

Reply:
Ah you are right. It is not needed on the paseo-local side. I'll remove.

Reply:
Removed